backend/src/routes/Call.ts

```diff
@@ -23,7 +23,10 @@ function CallRoutes(
       )
     );
     const totalDurationSeconds = twilioCalls
-      .map((twilioCall) => twilioCall.duration)
+      .map((twilioCall) => {
+        if (!twilioCall) return 0;
```

Comment:
`twilioCall: TwilioCall | null`. In the case of `null`, no need to increment total'

Reply:
Is this the style generated by Prettier? I thought prettier + eslint would enforce braces around conditionals

Reply:
duno leh. it didnt format for me. im adding curly manually